node/session.cpp

```diff
@@ -167,14 +156,13 @@ session::close()
         LOG(info) << "closing session";
 
         this->websocket->async_close(boost::beast::websocket::close_code::normal,
-            this->strand->wrap(
-                [self = shared_from_this()](auto ec)
+            [self = shared_from_this()](auto ec)
+            {
+                if (ec)
                 {
-                    if (ec)
-                    {
-                        LOG(error) << "failed to close websocket: " << ec.message();
-                    }
-                }));
+                    LOG(error) << "failed to close websocket: " << ec.message();
```

Comment:
Is this fatal? Should *never* happen, right?

Reply:
nope... it's probably already closed